code/modules/mining/machine_rigpress.dm

```diff
@@ -0,0 +1,75 @@
+/obj/machinery/mineral/rigpress
+	name = "RIG module press"
+	desc = "This machine converts certain items permanently into RIG modules."
+	icon = 'icons/obj/stationobjs.dmi'
+	icon_state = "coinpress0"
+	density = 1
+	anchored = 1.0
+	use_power = 1
+	idle_power_usage = 15
+	active_power_usage = 50
+	var/pressing
+
+/obj/machinery/mineral/rigpress/process()
```

Comment:
It processes already as a machine, does it not? That is why it inherits from the base machine process.